content/docs/installation/best-practice.md

```diff
@@ -22,6 +22,109 @@ are designed for backwards compatibility rather than for best practice or maximu
 You may find that the default resources do not comply with the security policy on your Kubernetes cluster
 and in that case you can modify the installation configuration using Helm chart values to override the defaults.
 
+## Isolate cert-manager on dedicated node pools
+
+cert-manager is a cluster scoped operator and you should treat it as part of your platform control plane.
+
+The cert-manager controller caches all the Secret resources of the cluster in memory,
```

Comment:
This is kinda weird and is definitely surprising.

Can it at least get an emoji/callout?

Also, is there a reason it can't only cache `Certificate` `Secrets` / `Secrets` it creates/manages?

Reply:
You're right, cert-manager [since release 1.12 does cache fewer Secrets](https://cert-manager.io/docs/releases/release-notes/release-notes-1.12/#lower-memory-footprint).
I've updated the words.